tests/foreman/api/test_capsulecontent.py

```diff
@@ -1633,3 +1633,50 @@ def test_positive_content_counts_blank_update(
         assert (
             counts is None or len(counts['content_view_versions']) == 0
         ), f"No content counts expected, but got:\n{counts['content_view_versions']}."
+
+    def test_positive_reclaim_space(
+        self,
+        target_sat,
+        module_capsule_configured,
+    ):
+        """Verify the reclaim_space endpoint spawns the Reclaim space task
+        and apidoc references the endpoint correctly.
+
+        :id: eb16ed53-0489-4bb9-a0da-8d857a1c7d06
+
+        :setup:
+            1. A registered external Capsule.
+
+        :steps:
+            1. Trigger the reclaim space task via API, check it succeeds.
+            2. Check the apidoc references the correct endpoint.
+
+        :expectedresults:
+            1. Reclaim_space endpoint spawns the Reclaim space task and it succeeds.
+            2. Apidoc references the correct endpoint.
+
+        :CaseImportance: Medium
+
+        :BZ: 2218179
+
+        :customerscenario: true
+        """
+        # Trigger the reclaim space task via API, check it succeeds
+        task = module_capsule_configured.nailgun_capsule.content_reclaim_space()
+        assert task, 'No task was created for reclaim space.'
+        assert (
+            'Actions::Pulp3::CapsuleContent::ReclaimSpace' in task['label']
+        ), 'Unexpected task triggered'
+        assert 'success' in task['result'], 'Reclaim task did not succeed'
+
+        # Check the apidoc references the correct endpoint
+        reclaim_doc = next(
+            method
+            for method in target_sat.apidoc['docs']['resources']['capsule_content']['methods']
+            if '/apidoc/v2/capsule_content/reclaim_space' in method['doc_url']
+        )
```

Comment:
It is not likely to happen, but this can raise a `StopIteration` exception if the given method is not in the `capsule_content` resource. I would say it is worth wrapping in `try/except` and raising a more meaningful exception.